app/services/usps_exporter.rb

```diff
@@ -1,22 +1,24 @@
 class UspsExporter
-  OTP_MAX_VALID_DAYS = 30
+  OTP_MAX_VALID_DAYS = Figaro.env.usps_confirmation_max_days.to_i
 
-  def initialize(csv_file_path)
-    @csv_file_path = csv_file_path
+  def initialize(psv_file_path)
```

Comment:
gotcha, so even though we were always separating with pipes, we decided to move away from 'csv' for more accurate naming?